systemtest/src/test/java/io/strimzi/systemtest/mirrormaker/MirrorMaker2ST.java

```diff
@@ -144,6 +152,25 @@ void testMirrorMaker2() {
                 .done();
         LOGGER.info("Looks like the mirrormaker2 cluster my-cluster deployed OK");
 
+        //deploying example clients for checking if mm2 will mirror messages with headers
+        KafkaClientsResource.consumerStrimzi(targetConsumerName, KafkaResources.plainBootstrapAddress(kafkaClusterTargetName), targetExampleTopic, MESSAGE_COUNT).done();
+        KafkaClientsResource.producerStrimzi(sourceProducerName, KafkaResources.plainBootstrapAddress(kafkaClusterSourceName), sourceExampleTopic, MESSAGE_COUNT)
```

Comment:
I think default config for delay between messages is 1000ms, maybe this is the problem why you want to increase the timeout? You can set it to 0ms to get max throughput.

Reply:
If you remember our conversation about this delay, I tried it to `1ms` (0ms is not allowed), `10ms` and `100ms` and the client acted real weird, as it wasn't able to finish the job. Maybe it was problem for `http-producer-client` and I definitely try that with `hello-world-producer/consumer`

Reply:
If you remember our conversation about this delay, I tried it to 1ms (0ms is not allowed), 10ms and 100ms and the client acted real weird, as it wasn't able to finish the job. Maybe it was problem for http-producer-client and I definitely try that with hello-world-producer/consumer

Reply:
Now I tried all the options that I mentioned above and the log doesn't even appear. Maybe it's again problem for `oc cluster up`, but I have to keep that `1000ms` delay and update the timeouts...

Reply:
I am quite sure 0 is working for kafka clients. We discussed bridge clients, which are implemented via vertx web client.

Reply:
So with 0 delay, not log is available but job is finished?

Reply:
Yes, I think you are right that 0ms delay will be working, gonna check it. But even tho it will be finished, the log will not appear. So we will need to have higher timeout for the single job finish